frontend/dockerfile/dockerfile_test.go

```diff
@@ -3023,16 +3035,25 @@ ADD *.tar /dest
 }
 
 func testDockerfileAddChownExpand(t *testing.T, sb integration.Sandbox) {
-	integration.SkipOnPlatform(t, "windows")
 	f := getFrontend(t, sb)
 
-	dockerfile := []byte(`
+	dockerfile := []byte(integration.UnixOrWindows(
+		`
 FROM busybox
 ARG group
 ENV owner 1000
 ADD --chown=${owner}:${group} foo /
 RUN [ "$(stat -c "%u %G" /foo)" == "1000 nobody" ]
-`)
+`,
+		`
+FROM nanoserver
+ARG group
+ENV owner=1000
+ADD foo \bar
+RUN if not exist \bar (exit 1)
+RUN if "%owner%"=="" if "%group%"=="" (exit 1)
```

Comment:
ownership is a lot different on Linux and Windows. You might need to revisit this. This is not the Windows equivalent dockerfile...

Reply:
I will leave out this test for now, looks like nanosever currently does not ship with user management tools like `net`. 
Default users in the container are already owners of newly created files.